lib/http/HttpClient_WinInet.cpp

```diff
@@ -32,6 +32,8 @@ class WinInetRequestWrapper
     HINTERNET              m_hWinInetRequest {nullptr};
     SimpleHttpRequest*     m_request;
     BYTE                   m_buffer[1024] {0};
+    DWORD                  m_bufferUsed {0};
+    std::vector<uint8_t>   m_bodyBuffer{};
```

Comment:
Nit: insert a trailing space after 'm_bodyBuffer' for consistency.

Reply:
fixed